rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -209,6 +211,16 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                 return true;
             }
         });
+
+        long lastSyncTimestamp = RDTApplication.getInstance().getContext().allSharedPreferences().fetchLastSyncDate(0);
+        if (lastSyncTimestamp > 0) {
+            Date lastSyncDate = new Date(lastSyncTimestamp);
+            String lblSync = getString(R.string.drawer_menu_item_sync) + ": %1$s, %2$s";
+            MenuItem syncMenuItem = menuNav.findItem(R.id.menu_item_sync);
+            syncMenuItem.setTitle(String.format(lblSync,
+                    new SimpleDateFormat("hh:mm a", org.smartregister.util.Utils.getDefaultLocale()).format(lastSyncDate),
+                    new SimpleDateFormat("MMM dd", org.smartregister.util.Utils.getDefaultLocale()).format(lastSyncDate)));
```

Comment:
Is it possible to convert the timestamp into a human readable date in one line?

Reply:
done

Reply:
https://stackoverflow.com/a/66424810/9782187